.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,8 +23,4 @@ Fixes #
 
 *
 
-<!--
-Helpful tips for screenshots:
-https://en.support.wordpress.com/make-a-screenshot/
--->
```

Comment:
Does this cause issues with the regex or something?

I'm asking because for third-party contributors, this might be relevant.. :thinking:

Reply:
I figured this is probably a well-spread skill now, especially for folks opening PRs, so just removed it as noise-reduction.